sql/src/test/java/io/crate/planner/fetch/FetchRewriterTest.java

```diff
@@ -1,201 +0,0 @@
-/*
```

Comment:
How about commenting these out / disabling them instead of removing? Suppose it doesn't matter since we don't merge to master yet.

Reply:
I think these tests won't be usable anymore in this form as they were very specific to how the `FetchRewriter` worked. The non-propagating fetch-logic is now already part of the `FetchOrEval` and `Collect` operators and that a fetch is planned is part of the `LogicalPlannerTest`.

I'll add more tests there when I add the fetch-propagation/before I open up the PR against master.